Project.toml

```diff
@@ -0,0 +1,25 @@
+name = "Clp"
+uuid = "5e71592d-7078-450d-8580-c595f51aa13b"
+authors = ["Mathieu Besançon <mathieu.besancon@gmail.com>"]
+repo = "https://github.com/JuliaOpt/Clp.jl.git"
+version = "0.6.3"
+
+[deps]
+BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
```

Comment:
We can remove it as we require Julia v1.0

Reply:
it seems MPB requires Compat for its tests

Reply:
Compat can be dropped when we don't support MPB